src/status_im2/contexts/wallet/account/bridge/style.cljs

```diff
@@ -0,0 +1,19 @@
+(ns status-im2.contexts.wallet.account.bridge.style)
+
+(defn container
+  [top]
+  {:flex       1
+   :margin-top top})
```

Comment:
I believe we should be using insets on the screens.cljs file instead. 
cc @flexsurfer

Reply:
@J-Son89 yes we should be using insets, fixed it